pkg/cluster/controllers/storage/provision.go

```diff
@@ -47,23 +47,28 @@ func (m *MinIOController) getMinIOProperties(minioInstance *minio.Tenant) (*goha
 	}
 
 	var (
-		endpoint string
-		scheme   string
+		minioHost string
+		endpoint  string
+		scheme    string
 
 		secure     = false
 		v4Auth     = true
-		skipVerify = false
+		skipVerify = true
 	)
 
 	if !m.HarborCluster.Spec.InClusterStorage.MinIOSpec.Redirect.Disable {
-		scheme, endpoint, err = GetMinIOHostAndSchema(m.HarborCluster.Spec.ExternalURL)
-		if err != nil {
-			return nil, err
+		scheme = "https"
+		if minioHost = m.HarborCluster.Spec.InClusterStorage.MinIOSpec.Host; minioHost == "" {
+			scheme, minioHost, err = GetMinIOHostAndSchema(m.HarborCluster.Spec.ExternalURL)
```

Comment:
I don't think we need to auto-generate the host from the `ExternalURL ` if the `host` is not set. See my above comment, the `host` should be required, not `optional`.